FileManager1.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <experimental/filesystem>
+#include <gtest/gtest.h>
+
+class FileManager 
+{ 
+    //Declaring the private attributes of the FileManager Class
+    private:
+    std::fstream fs; 
+    const int BlOCK_SIZE = 100; 
+
+    public:
+    //Default Constructor
+    FileManager()
+    {
+
+    }   
+
+    bool open(std::string filename)
+    {
+        fs.open(filename);
+        if (fs.is_open())
+        {
+            return true; 
+        }
+        return false; 
+    }
+
+    bool close()
+    {
+        fs.close();
+
+        return true; 
+    }
+
+    bool write(std::string filename, const std::string& data)
+    {
+        std::ofstream outfile = std::ofstream(filename);
+        outfile << data << std::endl; 
+        outfile.close();
+        return 0; 
+    }
+
+    std::string& getNextBlock(std::string filename)
```

Comment:
I like this idea of getting blocks of data -- I'll make sure to incorporate this in my code during the next push to my branch